gboard_autopsy.py

```diff
@@ -0,0 +1,123 @@
+import inspect
+import os
+
+from org.sleuthkit.autopsy.casemodule import Case
+from org.sleuthkit.autopsy.casemodule import NoCurrentCaseException
+from org.sleuthkit.autopsy.coreutils import Logger
+from org.sleuthkit.autopsy.ingest import IngestModuleFactoryAdapter
+from org.sleuthkit.autopsy.ingest import DataSourceIngestModule
+from org.sleuthkit.autopsy.ingest import IngestModule
+from org.sleuthkit.autopsy.ingest import IngestServices
+from org.sleuthkit.datamodel.blackboardutils import ArtifactsHelper
+from org.sleuthkit.datamodel import TskCoreException, TskDataException
+from org.sleuthkit.autopsy.casemodule.services import Blackboard
+from org.sleuthkit.autopsy.ingest.IngestModule import IngestModuleException
+
+from java.util.logging import Level
+
+class GboardDataSourceIngestModuleFactory(IngestModuleFactoryAdapter):
+
+    moduleName = "GBoard4A"
```

Comment:
I assume this is used internally by the autopsy module loader. I kept it because the example suggest to have it the same way: https://github.com/sleuthkit/autopsy/blob/develop/pythonExamples/dataSourceIngestModule.py#L63

Reply:
Ah ok, it's pretty much identical to the example, I see. But it is weird because it says `Will be shown in module list, logs, etc.`, making it seem that it is a read-only property.